src/shared/services/dom-util.service.ts

```diff
@@ -32,6 +32,41 @@ export class DomUtilService {
       if (firstInput) {
         firstInput.focus();
         firstInput.select();
+      }else {
+        firstInput = el.querySelector('div.btn-group') as HTMLInputElement;
+        if (firstInput) {
+          // .open is used for opening the dropdownmenu
+          firstInput.classList.add('open');
+          firstInput = el.querySelector('div.btn-group input') as HTMLInputElement;
+          firstInput.focus();
+          firstInput.select();
+        }
+      }
+    }
+  }
+
+  focusInputByTabindex(index: number) {
```

Comment:
can't you get the `id` with tab index since we have all `id`s sorted in an array, then select by id. This could be be faster than this.